suites/squid/interop/test-ceph-sanity.yaml

```diff
@@ -81,75 +81,75 @@ tests:
       module: test_cephadm.py
       name: Deploy cluster using cephadm
 
-  - test:
-      abort-on-fail: true
-      config:
-        command: add
-        id: client.1
-        node: node6
-        install_packages:
-          - ceph-common
-          - rbd-nbd
-          - jq
-          - fio
-        copy_admin_keyring: true
-      desc: Configure the RGW,RBD client system
-      destroy-cluster: false
-      module: test_client.py
-      name: configure client
+  # - test:
```

Comment:
why these tests are commented ?

Reply:
i have commented it for testing, i will uncomment update PR